src/static/hyperhtml/examples/example/react/not-primitive-props/hyper.js

```diff
@@ -0,0 +1,26 @@
+customElements.define(
+  'h-welcome',
+  class HyperWelcome extends HTMLElement {
+    constructor(...args) {
+      super(...args);
+      this.html = hyperHTML.bind(this);
+    }
+
+    get user() { return this._user; }
+    set user(value) {
+      this._user = value;
+      this.render();
+    }
+
+    connectedCallback() { this.render(); }
+    render() {
+      return this.html`<h1>Hello, ${this._user.name}</h1>`;
```

Comment:
Witch Custom Element you don't need to return anything. Just populate the element via `this.html` and drop the return.

Reply:
Yup I know this, but I'd prefer to keep the return to make it feel more "pure" and less side-effectish.